cli/JSXV4UPGRADE.md

```diff
@@ -0,0 +1,44 @@
+**JSX V4 upgrade path**
+
+1. No upgrading rescript-react && no changes in bsconfig.json
+
+   JSX V3 by default.
+
+2. No upgrading rescript-react && new configuration added in bsconfig.json
+
+   a. JSX V3
+
+   ```json
+   "react": {
+     "jsx": 3,
+     "runtime": "classic"
+   }
+   ```
+
+   The `react.runtime` affects nothing, no matter of `"classic"` or `"automatic"`.
```

Comment:
If the `react.runtime` flag "affect nothing", why is it there?

Reply:
Good catch. It doesn't have to be there, but as an aspect of schema, it works with `react.jsx: 4`

Reply:
Thanks!